Adobe/SmartGroupTemplate.xml

```diff
@@ -0,0 +1,27 @@
+<computer_group>
+	<name>%group_name%</name>
+	<is_smart>true</is_smart>
+	<criteria>
+		<criterion>
+			<name>Application Title</name>
+			<priority>0</priority>
+			<and_or>and</and_or>
+			<search_type>is</search_type>
+			<value>%JSS_INVENTORY_NAME%</value>
+		</criterion>
+		<criterion>
+			<name>Application Version</name>
+			<priority>1</priority>
+			<and_or>and</and_or>
+			<search_type>is not</search_type>
+			<value>%VERSION%</value>
```

Comment:
Curious why `VERSION` and not the `version` you're extracting using `CreativeCloudVersioner`? Is there some additional translation that the JSSImporter does that I'm missing here?